install-conda-dependencies.sh

```diff
@@ -6,7 +6,7 @@ echo "==== Installing Conda packages for $(which conda)"
 
 conda info --envs
 
-conda install --yes pocl pyvisfile
+conda install --yes pocl pyvisfile clinfo pyopencl islpy cffi
```

Comment:
Why do we need `cffi`? Neither pyopencl nor islpy depend on it. (For islpy, that's a pretty recent change.)

Reply:
Conda's islpy depends on cffi, and I have seems some version mismatches with cffi when installing other packages later. But installing it here doesnt seem to make a difference, so will remove it.

Reply:
Ah, you're right. Git master doesn't, but the last release still does. I was going to roll a release once the de-cffi-fied version had been around for a bit, so... now.

Reply:
https://pypi.org/project/islpy/2020.2/

Reply:
Should propagate to conda-forge pretty soon (though it'll likely need manual intervention because of the changed dependencies).

Reply:
https://github.com/conda-forge/islpy-feedstock/pull/15